numpy/linalg/linalg.py

```diff
@@ -243,7 +243,7 @@ def tensorsolve(a, b, axes=None):
 
     It is assumed that all indices of `x` are summed over in the product,
     together with the rightmost indices of `a`, as is done in, for example,
-    ``tensordot(a, x, axes=b.ndim)``.
+    ``tensordot(a, x, axes=x.ndim)``.
```

Comment:
The documentation and the implementation refers to `tensorsolve(a, b)`. I think axes can be any arbitrary tuple. This line is wrong, but I don't think the fix is correct.

Reply:
Well I think the `tensordot` refers to how to contract a and x, because of the "as is done in, for example" part of the sentence.